assem16399/Diva
# Feature/home screen/diva 27 home data

Jira Ticket: [DIVA-27](https://aasemhany.atlassian.net/browse/DIVA-27)

Description 📖
Implementing homeScreen Feature data layer (Models, ApiService, ApiRepo)

How to Test 📋
check the Home Feature Data Layer folder

[DIVA-27]: https://aasemhany.atlassian.net/browse/DIVA-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
